typing_extensions/src_py2/test_typing_extensions.py

```diff
@@ -377,6 +377,44 @@ def test_annotated_in_other_types(self):
         self.assertEqual(X[int], List[Annotated[int, 5]])
 
 
+class TypeAliasTests(BaseTestCase):
```

Comment:
Could you add some positive tests? I've forgotten the syntax we agreed upon and there's no hint in this PR of what it should be.